src/effects/RealtimeEffectState.h

```diff
@@ -87,6 +91,9 @@ class RealtimeEffectState
    //! Main thread cleans up playback
    bool Finalize() noexcept;
 
+   //! Notify all subscribers of a state change
+   void PublishStateUpdate(RealtimeEffectStateChange state);
```

Comment:
You can do `using Observer::Publisher<RealtimeEffectStateChange>::Publish`, but I don't yet understand why this function is public

Reply:
Its also called inside EffectUIHost::OnEnable to signal back to the realtime side panel that there has been a state change.

Reply:
Done.